.github/workflows/run-cctp-tests.yml

```diff
@@ -0,0 +1,82 @@
+name: PR approved workflow
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_review:
+    types:
+      - submitted
+  push:
+    branches: ["master"]
+  merge_group:
+
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-cctp
+  cancel-in-progress: true
+
+env:
+  NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
+  NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID }}
+  THE_GRAPH_NETWORK_API_KEY: ${{ secrets.THE_GRAPH_NETWORK_API_KEY }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  check-files:
+    name: Check files
+    runs-on: ubuntu-latest
+    outputs:
+      run_tests: ${{ steps.check-files.outputs.run_tests }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/check-files
+        id: check-files
+
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    needs: [check-files]
+    if: |
+      (needs.check-files.outputs.run_tests == 'true' && ${{ !contains(github.event.pull_request.title, 'hotfix') }}) ||
+      (contains(github.event.comment.html_url, '/pull/') && contains(github.event.comment.body, '/test'))
```

Comment:
do we want to use `/test` or `/cctp-test` or `/run-cctp-test` or `Run CCTP test` (case-insensitive)?